src/dune_rules/melange_stanzas.ml

```diff
@@ -47,6 +48,15 @@ module Emit = struct
     t
 
   let decode =
+    let extension_field name =
+      let+ field = field_o name (located string) in
```

Comment:
You can use `field` and set `~default`.

Reply:
https://github.com/ocaml/dune/pull/6457/commits/365e79e4b45224bd9c32a637c618d84a7575137a